src/xrGame/string_table.cpp

```diff
@@ -82,21 +85,24 @@ void CStringTable::FillLanguageToken()
 void CStringTable::SetLanguage()
 {
     if (LanguageID != std::numeric_limits<u32>::max())
-        pData->m_sLanguage = languagesToken.at(LanguageID).name;
+        if (languagesToken.size() > LanguageID)
+            pData->m_sLanguage = languagesToken.at(LanguageID).name;
```

Comment:
I will submit my refactoring changes, soon. I hope they won't collide with this..